deploy/osd-storage/cluster-storage-gcp-standard.yaml

```
@@ -0,0 +1,13 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: standard
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "false"
+provisioner: kubernetes.io/gce-pd
+parameters:
+  type: pd-standard
```

```suggestion
  type: pd-ssd
```
We already have pd-standard, we want to add pd-ssd right?